pkg/controllers/multicluster/serviceexport_controller.go

```diff
@@ -225,6 +225,13 @@ func (r *ServiceExportReconciler) extractEndpoints(ctx context.Context, svc *v1.
 		return nil, err
 	}
 
+	var serviceType model.ServiceType
+	if svc.Spec.ClusterIP == "None" {
```

Comment:
Will `svc.Spec.ClusterIP` always be None for headless service? Or it can be empty string too?

Reply:
It will always be None for headless services according to [spec](https://pkg.go.dev/k8s.io/api@v0.24.2/core/v1#ServiceSpec).